packages/OverflowMenu/src/OverflowMenu.js

```diff
@@ -43,12 +46,13 @@ const defaultProps = {
   isOpen: null,
   onClose: null,
   zIndex: zValue(1),
+  maxHeight: null,
```

Comment:
Should we leave maxHeight undefined if it's not specified? or is there a default value that we should use

Reply:
`null` is usually preferred. Ideally these longer lists of props would be alphabetized though, just a good habit to make it easier to find something (or verify if something is missing).

Reply:
Some of our components allow `number` or `string` for these kinds of values (I know I recently updated the `<Textarea>` to allow this).  That makes it a little more flexible, for instance, I could set the `maxHeight` to `45vh` – which actually might make a nice default value instead of `null`.  

Probably worth discussing the default behaviour with @sunminsarahlee